Garrett-Iannuzzi/BYOB
# Post requests

### What does this PR do?
- Adds a post request for adding a metro area
- Adds a post request for adding a title
- Updates the RAEDME to reflect the two post requests

### How to test?
- Update your branch to reflect the post requests
- Use postman to follow the correct endpoint /api/v1/metros for adding a metro
- Use postman to follow the correct endpoint ```/api/v1/titles/:metro_id/titles`` for adding a title
- To test correctly enter an new metro to see the 201 level response, the response should be an object with a key of metro and the values from the request.body
- To test correctly enter an new title to see the 201 level response, the response should be an object with a key of title and the values from the request.body
- Both will have a 422 level error if there is missing information

### Issues
This PR does not close any issues
Changes have been updated in the project board

### Notes
- This PR will need to undergo git rebase to merge

Thanks!
@Garrett-Iannuzzi
